### DISCUSSION TITLE
Running "Run Background" in Android

### DISCUSSION THREAD

**Discussion by ascoider** [asus4/tf-lite-unity-sample#278] [Q&A] [answered]
Hi! This is a great work! I'm currently using the BlazePose example in a custom project. It runs prefectly in Android when the option "Run background" is deativated, but when is it activated, it doesn't render any joint or skeleton, as ver pose return always null. This, however, doesn't happen when running the scene in editor mode in unity. Is there something I am missing? I am interested if it is possible to make it work as it improves drastically the FPS.
Thanks!

**Comment by asus4** [comment, marked as answer]
First post in discussions 🎉 

Are you running it on the Unity 2021 LTS? If so, please refer to https://github.com/asus4/tf-lite-unity-sample/issues/263

You need to modify AndroidManifest.xml to enable OpenCL acceleration at the API level 31.

Performance-wise, 
- Minimum latency: Invoking it in the same thread
- Minimum impact on the game FPS: Invoking it in the background-thread, instead of 1~2 frames latency.

**Reply by ascoider** [reply]
Thanks! It worked perfectly.
